velox/exec/OutputBufferManager.cpp

```diff
@@ -183,4 +183,9 @@ bool OutputBufferManager::isOverutilized(const std::string& taskId) {
   return false;
 }
 
+OutputBufferInfo OutputBufferManager::getInfo(const std::string& taskId) {
+  auto buffer = getBuffer(taskId);
```

Comment:
No, `getBuffer()` would never return `nullptr`. `getBufferIfExists()` would return `nullptr`, but as we are getting the stats of a task's outputBuffer, the outputBuffer should exist.

Reply:
Thank you for your explanation, `getBuffer()` does not return `nullptr`, but it may throw an exception.

No offense, I just wonder about the better behavior of `getStats()` if this "task" no longer exists (this must happen, right? That is, after the buffer ends, another thread calls the function)?

If an exception is thrown, the caller needs additional `try...catch...`.

At the same time, I noticed that another API for obtaining information (`getUtilization()`) will face the same situation, but it will not throw an exception.

Reply:
Many existing interfaces including `updateNumDrivers()`, `enqueue()`, `noMoreData()`, `isFinished()` use the `getBuffer()` function to get the buffer without checking. For these interfaces, the buffer **should** exist, otherwise an exception is expected. I think this is by design. 

If you have doubts on the existing `OutputBufferManager`'s design, maybe you should open another issue to discuss it rather than discuss here. This PR is not for it.